jumanji/environments/packing/bin_pack/viewer.py

```diff
@@ -274,3 +274,85 @@ def _get_used_volume(self, state: State) -> float:
             if placed
         )
         return used_volume
+
+
+class ConstrainedBinPackViewer(BinPackViewer):
```

Comment:
What's the difference compared to the usual one here?

Reply:
The methods `create_entities` and `add_overlay` and `_get_used_volume` have to be changed

Reply:
Could you add a docstring explaining this?

Reply:
I added a docstring at the beginning of the class definition.

Reply:
I can't see the docstring is the latest version pushed?